examples/post_training_quantization/torch/mobilenet_v2/main.py

```diff
@@ -138,22 +140,15 @@ def transform_fn(data_item: Tuple[torch.Tensor, int]) -> torch.Tensor:
 # item and prepare model input data. The quantize method uses a small subset
 # (default: 300 samples) of the calibration dataset.
 
-calibration_dataset = nncf.Dataset(val_data_loader, transform_fn)
-torch_quantized_model = nncf.quantize(
-    torch_model,
-    calibration_dataset,
-    advanced_parameters=nncf.AdvancedQuantizationParameters(disable_bias_correction=True),
-)
+calibration_dataset = nncf.Dataset(val_data_loader, partial(transform_fn, device=device))
+torch_quantized_model = nncf.quantize(torch_model, calibration_dataset)
```

Comment:
What about accuracy numbers for the `test_examples` since bias correction was enabled?

Reply:
Thanks for the comment! I submitted new reference metrics.